src/main.ts

```diff
@@ -22,7 +23,18 @@ export default function main(port: number = Config.port) {
   ) => {
     response.setHeader("content-type", "text/plain;charset=utf8");
     response.writeHead(200, "OK");
-    response.end("Olá, Hola, Hello!");
+    const url = request.url;
+
+    if (url === "/" && request.method === "POST") {
+      io.emit("state", "refresh");
+      response.write("emit refresh");
+      response.end();
+    }
```

Comment:
It would be nice to customize the URL to avoid intruders:

```suggestion
    if (url === process.env.secretUrl ?? "/" && request.method === "POST") {
      io.emit("state", "refresh");
      response.write("emit refresh");
      response.end();
    }
```

Do you think that adding a test for this functionality is possible?

Reply:
off course!